src/uct/ib/base/ib_iface.c

```diff
@@ -276,7 +280,9 @@ void uct_ib_address_pack(const union ibv_gid *gid, uint16_t lid,
         *(uint16_t*) ptr = lid;
         ptr              = UCS_PTR_BYTE_OFFSET(ptr, sizeof(uint16_t));
 
-        if (is_global_addr) {
+        if (is_global_addr) { /* we need this flag check to ensure that
+                               * gid->global.subnet_prefix is accessible
+                               * (initialized by rdmacm) */
```

Comment:
revise the comment. before the 'if':
`/* if is_global_addr is false, *gid is undefined */`